src/ServiceManagerBase.sol

```diff
@@ -62,6 +71,48 @@ abstract contract ServiceManagerBase is OwnableUpgradeable, ServiceManagerBaseSt
     ) internal virtual onlyInitializing {
         _transferOwnership(initialOwner);
         _setRewardsInitiator(_rewardsInitiator);
+        isStrategiesMigrated = true;
+    }
+
+    /// TODO: natspec
+    function addStrategiesToOperatorSet(
+		uint32 operatorSetID,
+		IStrategy[] calldata strategies
+	) external onlyStakeRegistry {
+        _operatorSetManager.addStrategiesToOperatorSet(
+            operatorSetID,
+            strategies
+        );
+    }
+
+    /// TODO: natspec
+    function removeStrategiesFromOperatorSet(
+		uint32 operatorSetID,
+		IStrategy[] calldata strategies
+	) external onlyStakeRegistry {
+        _operatorSetManager.removeStrategiesFromOperatorSet(
+            operatorSetID,
+            strategies
+        );
+    }
+
+    function migrateStrategiesToOperatorSets() external {
+        uint8 quorumCount = _registryCoordinator.quorumCount();
```

Comment:
`require(!isStrategiesMigrated)`

Reply:
TODO: potentially a way to do this via a view function on OSM

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/blob/51ff1682890f537915b90abfb5074daedd61949e/src/contracts/core/AVSDirectoryStorage.sol#L45